airgun/widgets.py

```diff
@@ -2378,3 +2422,33 @@ def fill(self, value):
 
     def read(self):
         return self.selected
+
+
+class SatPatternflyTable(BasePatternflyTable, Table):
+    def __init__(
+        self,
+        parent,
+        column_widgets=None,
+        assoc_column=None,
+        rows_ignore_top=None,
+        rows_ignore_bottom=None,
+        top_ignore_fill=False,
+        bottom_ignore_fill=False,
+        logger=None,
+    ):
+        self.component_type = "PF4/Table"
```

Comment:
This is used for OUIA type widgets, why use the non-OUIA base classes?

Reply:
Ack.  Will try that.  Prior to this `self.component_id = component_id` didn't have static ouia-id so it was constantly failing in recognizing the table and so we switched over to not rely on that.  Will switch it back and retry.

Reply:
It seems this is also used in `ouia.py` so I have deleted mine.